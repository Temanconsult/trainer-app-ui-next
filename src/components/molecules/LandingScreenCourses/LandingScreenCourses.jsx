import React from 'react'
import { Row } from 'react-bootstrap'

import CourseInfoCard from '../../atoms/CourseInfoCard'
import courses from '../../../utils/courses'

const LandingScreenCourses = () => {
  return (
    <Row className="landing-screen-courses mt-4 mx-auto ">
      <h2>Courses</h2>
      <Row className="courses">
        {courses.map((course) => {
          return <CourseInfoCard key={course.title} course={course} />
        })}
      </Row>
    </Row>
  )
}

export default LandingScreenCourses
