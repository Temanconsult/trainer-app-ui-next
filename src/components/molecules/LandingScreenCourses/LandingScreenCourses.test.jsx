import React from 'react'
import { render } from '@testing-library/react'
import LandingScreenCourses from '.'

describe('LandingScreenCourses', () => {
  it('creates landing screen courses', () => {
    const { asFragment } = render(<LandingScreenCourses />)
    expect(asFragment()).toMatchSnapshot()
  })
})
