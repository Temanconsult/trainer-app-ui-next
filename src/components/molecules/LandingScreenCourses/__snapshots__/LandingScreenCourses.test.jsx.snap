// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LandingScreenCourses creates landing screen courses 1`] = `
<DocumentFragment>
  <div
    class="landing-screen-courses mt-4 mx-auto  row"
  >
    <h2>
      Courses
    </h2>
    <div
      class="courses row"
    >
      <div
        class="mb-4 mr-md-2 mr-lg-4 mr-sm-0 card bg-info text-white"
        style="max-width: 30rem;"
      >
        <div
          class="card-body"
        >
          <img
            class="card-img-top"
            src="file mock"
          />
          <div
            class="card-title h5"
          >
            Scrum Master
          </div>
          <p
            class="card-text"
          >
            A scrum master is the facilitator for an agile development team. Scrum is a methodology that allows a team to self-organize and make changes quickly, in accordance with agile principles. The scrum master manages the process for how information is exchanged.
          </p>
        </div>
      </div>
      <div
        class="mb-4 mr-md-2 mr-lg-4 mr-sm-0 card bg-light text-dark"
        style="max-width: 30rem;"
      >
        <div
          class="card-body"
        >
          <img
            class="card-img-top"
            src="file mock"
          />
          <div
            class="card-title h5"
          >
            JavaScript
          </div>
          <p
            class="card-text"
          >
            JavaScript (JS) is a lightweight, interpreted, or just-in-time compiled programming language with first-class functions. While it is most well-known as the scripting language for Web pages, many non-browser environments also use it, such as Node.js, Apache CouchDB and Adobe Acrobat.
          </p>
        </div>
      </div>
      <div
        class="mb-4 mr-md-2 mr-lg-4 mr-sm-0 card bg-info text-white"
        style="max-width: 30rem;"
      >
        <div
          class="card-body"
        >
          <img
            class="card-img-top"
            src="file mock"
          />
          <div
            class="card-title h5"
          >
            DevOps
          </div>
          <p
            class="card-text"
          >
            DevOps is a set of practices that combines software development and IT operations. It aims to shorten the systems development life cycle and provide continuous delivery with high software quality.
          </p>
        </div>
      </div>
      <div
        class="mb-4 mr-md-2 mr-lg-4 mr-sm-0 card bg-light text-dark"
        style="max-width: 30rem;"
      >
        <div
          class="card-body"
        >
          <img
            class="card-img-top"
            src="file mock"
          />
          <div
            class="card-title h5"
          >
            Project Management
          </div>
          <p
            class="card-text"
          >
            Project management is the practice of leading the work of a team to achieve goals and meet success criteria at a specified time.
          </p>
        </div>
      </div>
      <div
        class="mb-4 mr-md-2 mr-lg-4 mr-sm-0 card bg-info text-white"
        style="max-width: 30rem;"
      >
        <div
          class="card-body"
        >
          <img
            class="card-img-top"
            src="file mock"
          />
          <div
            class="card-title h5"
          >
            Data Science
          </div>
          <p
            class="card-text"
          >
            Data science is an inter-disciplinary field that uses scientific methods, processes, algorithms and systems to extract knowledge and insights from many structural and unstructured data.
          </p>
        </div>
      </div>
      <div
        class="mb-4 mr-md-2 mr-lg-4 mr-sm-0 card bg-light text-dark"
        style="max-width: 30rem;"
      >
        <div
          class="card-body"
        >
          <img
            class="card-img-top"
            src="file mock"
          />
          <div
            class="card-title h5"
          >
            Cloud
          </div>
          <p
            class="card-text"
          >
            Cloud computing is the on-demand availability of computer system resources, especially data storage and computing power, without direct active management by the user.
          </p>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
